plugins/modules/docker_compose_v2_run.py

```diff
@@ -253,7 +252,10 @@
     common_compose_argspec_ex,
 )
 
-from ansible_collections.community.docker.plugins.module_utils.util import sanitize_labels
+from ansible_collections.community.docker.plugins.module_utils.util import (
+    sanitize_labels,
```

Comment:
The correct way to fix this is to create a new function `sanitize_list()` in `ansible_collections.community.docker.plugins.module_utils.util` with a similar interface (and implementation) to `sanitize_labels()`, except that it processes a list and not dictionary values.

(And I'm really looking forward to getting rid of Python 2.7, so we can add typing hints to prevent such bugs...)

Reply:
Thank you for the hints. It should be done now.